tsconfig.base.json

```diff
@@ -1,20 +1,21 @@
 {
-  "compileOnSave": false,
   "compilerOptions": {
-    "rootDir": ".",
-    "sourceMap": true,
-    "declaration": false,
+    "lib": ["ES2020", "dom"],
+    "module": "ES2020",
+    "target": "ES2020",
     "moduleResolution": "node",
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "importHelpers": true,
-    "target": "es2015",
-    "module": "esnext",
-    "lib": ["es2017", "dom"],
-    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "strict": true,
+    "strictNullChecks": true,
+    "resolveJsonModule": true,
     "skipDefaultLibCheck": true,
-    "baseUrl": ".",
-    "paths": {}
+    "skipLibCheck": true,
+    "declaration": true,
+    "inlineSourceMap": true,
+    "paths": {
+      "@remix-storage/bodyparser": ["./packages/bodyparser/src/index.ts"]
```

Comment:
This should work but Remix doesn't support `@something` as a valid `path`. So we have to hack remix apps with `~` like this: 
```json
"paths": {
  "~remix-storage/bodyparser": ["../../packages/bodyparser/src/index.ts"]
}
```